_posts/2024-05-08-eoss-recap.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Embedded Open Source Summit 2024 Recap
+description:
+  A discussion of our favorite talks from EOSS 2024, including those in the
+  Embedded Linux Conference and the Zephyr Developer Summit
+author: tsarlandie
+tags: [conferences]
+---
+
+April marked the return of the Embedded Open-Source Summit, this year in
+Seattle. I was lucky enough to be able to attend and split my time between the
+Memfault booth in the exposition hall and many of the captivating presentations.
+Since
+[the videos have just been published](https://www.youtube.com/playlist?list=PLbzoR-pLrL6q2lcVcQ8s1uVNxde0aVdp3)
+on the the Linux Foundation’s YouTube account, we thought it would be a good
+time to highlight some of the talks and give you a quick summary which will,
+hopefully, inspire you to go watch them!
+
+<!-- excerpt start -->
+
+We cover the talks I was able to see in person, as well as some talks since by
+my colleagues since they were posted. Obviously this is just our little biased
```

Comment:
```suggestion
We cover the talks I was able to see in person, as well as some talks seen by
my colleagues since they were posted. Obviously this is just our little biased
```